ui/widgets/checkbox.cpp

```diff
@@ -908,6 +911,14 @@ void RadiobuttonGroup::setValue(int value) {
 	for (const auto button : _buttons) {
 		button->handleNewGroupValue(_value);
 	}
+	if (base::ScreenReaderState::Instance()->activeValue()) {
+		for (const auto button : _buttons) {
+			button->setFocusPolicy((button->_value == _value)
+				? Qt::StrongFocus
+				: Qt::NoFocus);
+		}
```

Comment:
Instead of the second loop, the code could be moved to `Radiobutton::handleNewGroupValue`